tests/vmi_configuration_test.go

```diff
@@ -1290,6 +1292,7 @@ var _ = Describe("Configurations", func() {
 				_, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(Vmi)
 				Expect(err).ToNot(HaveOccurred())
 				node1 := tests.WaitForSuccessfulVMIStart(Vmi)
```

Comment:
Ha, `node1` needs to be definitely renamed to `node`

Reply:
done